src/main/java/frc/robot/subsystems/vision/VisionConstants.java

```diff
@@ -1,9 +1,66 @@
 package frc.robot.subsystems.vision;
 
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.util.Units;
+import org.photonvision.PhotonCamera;
 
 public final class VisionConstants {
+  public enum Limelight {
```

Comment:
I like the enum. I was gonna tell you to change it to `Limelights` but I'm pretty sure this is correct convention lol.